src/store/token-dashboard/slice.ts

```diff
@@ -40,17 +40,32 @@ export type ModalState = Nullable<
   | { stage: 'DISCORD_CODE' }
 >
 
-export type AssociatedWallets = string[]
+export type AssociatedWallets = {
+  address: string
+  balance: BNWei
+  collectibleCount: number
+}[]
 
-export type ConfirmRemoveWalletAction = PayloadAction<{ wallet: WalletAddress }>
+export type ConfirmRemoveWalletAction = PayloadAction<{
+  wallet: WalletAddress
+  chain: Chain
+}>
 
+export type Chain = 'sol' | 'eth'
```

Comment:
i know mike likes the string options here, but an enum would let you switch statement the reducers a bit more cleanly 🤷